build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
If we can't crossgen non-Windows components on Windows, it might be easier as a stop-gap to have each Helix job attempt to crossgen Core_Root .dll's before executing tests